tzrec/models/model.py

```diff
@@ -238,8 +238,8 @@ def forward(
         return self.model.predict(batch)
 
 
-class ExportWrapperAOT(ScriptWrapper):
-    """Model inference wrapper for aot export."""
+class CudaScriptWrapper(ScriptWrapper):
```

Comment:
CudaExportWrapper is better, it may not be a scripted model.

Reply:
ok